src/sentry/lang/javascript/processor.py

```diff
@@ -578,15 +614,16 @@ def expand_frames(self, frames, release):
                     })
 
                 # Store original data in annotation
+                # HACK(dcramer): we stuff things into raw which gets popped off
+                # later when adding the raw_stacktrace attribute.
+                raw_frame = frame.to_json()
                 frame.data = {
-                    'orig_lineno': frame.lineno,
```

Comment:
What about all the old data that has this stored? Are we just bailing on all of it in the UI?